env-local.env.sample

```diff
@@ -1,12 +1,28 @@
 ## The variables below are required for the production enviroment.
 ## Some might be also required for the local environment.
 
+# Assign this to the adequate Docker context.
+# This will be used in the `docker context` command.
+# To read more about how to create and use a Docker context, visit
+# https://docs.docker.com/engine/reference/commandline/context/
+# The default value for a Docker setup is that the local context
+# is called `default`.
+# When deploying to a remote server, first create a Docker context
+# manually, and then reference it here.
```

Comment:
Convenient with this inline doc here. There is a bit of overlap with what we have in the wiki. Do we want to make that page public, perhaps under `.github` and link to that instead?

Reply:
We do have some stuff in the wiki, but if we would move it to a public location I'd rather we do it in a planned way, by having a dedicated issue and a discussion about it. That's why it feels easier to me right now to reference some external documentation here, like a guide on Medium or something.